Piwik.Tracker/PiwikTracker.cs

```diff
@@ -1280,6 +1279,7 @@ protected bool LoadVisitorIdCookie()
             {
                 _visitCount = long.Parse(parts[2]);
             }
+            // todo _currentVisitTs is never used (was earlier assigned with null in constructor)

```

Comment:
currentVisitTs is only set by the library, it is not used anywhere in the library, that is why I thought it was forgotten to be used. I did not know that this value may be interesting for the consumer. Of course we can introduce a public getter for currentVisitTs, eg. public CurrentVisitTs {get;}. What do you mean by "Is it bad practice if we keep the code as is?"

> 

>